[Meta] Video Player Documentation - Move AV1 from Video to Container Section (#1416)
jellyfin/Swiftfin: Documentation/players.md

| Video Codec | Swiftfin (VLCKit) | Native (AVKit) |
|-------------|-------------------|----------------|
| **AV1**     | ✅                | 🟡 Limited support |
| **H.264**   | ✅                | ✅             |
| **H.265**   | ✅                | ✅             |
| **MPEG-2**  | ✅                | ❌             |
| **MPEG-4**  | ✅                | ✅             |
| **VP8**     | ✅                | ❌             |
| **VP9**     | ✅                | ❌             |

> - AV1 is disabled by default but can be enabled for Native (AVKit) using Custom Device Profiles. Enabling AV1 may result in a [poor experience for non-A17 SOCs and higher](https://en.wikipedia.org/wiki/Apple_A17).

Reviewer: ```suggestion
- AV1 is disabled by default but can be enabled for Native (AVKit) using Custom Device Profiles. Enabling AV1 may result in a [poor experience for SOCs prior to A17](https://en.wikipedia.org/wiki/Apple_A17).
```
